app/src/main/java/in/co/rajkumaar/amritarepo/faq/ExamsFAQActivity.java

```diff
@@ -62,11 +61,13 @@ public void onPageFinished(WebView view, String url) {
     }
 
     private void getFAQ() {
+        final Document[] htmlDoc = new Document[1];
+        final String[] finalHtml = new String[1];
```

Comment:
This is bad coding standards. Use a stringBuilder variable and append. And let htmlDoc be just an element, don't convert to an array.